City-of-Helsinki/drupal-hdbt
# UHF-X Update svg-sprite

# Update svg-sprite

## What was done
* Updated glob dependency
* Updated Node.js to v16.15.0
* Update SVG Sprite node module to use specific commit https://github.com/svg-sprite/svg-sprite/commit/ae913149d57ed602bb34bc76f6bfad3569360fbc

## How to test
* Checkout this branch
* Run `nvm use && npm i && npm run build`
* Check that the sprite compilation still works
* Also check that icons work in your local instance
